src/slice.rs

```diff
@@ -146,6 +147,7 @@ impl SliceInfoElem {
     }
 }
 
+#[cfg(feature = "std")]
```

Comment:
I don't think this change is necessary either (since `core::fmt::{Formatter, Result}` and `core::{write, stringify}` exist), and it would be nice to continue supporting this implementation in `no_std` environments.

Reply:
Thanks, I update my PR.